sdk/iOS/test/MSCoreDataStoreTests.m

```diff
@@ -20,6 +20,9 @@ @interface MSCoreDataStoreTests : XCTestCase {
 
 @implementation MSCoreDataStoreTests
 
+// Table used for running tests
+NSString * const TableName = @"TodoItem";
```

Comment:
Didn't realize defining a variable inside an implementation block does not make it an class member. Added static now.